site/content/src/test/scala/exercises/stdlib/AssertsSpec.scala

```diff
@@ -0,0 +1,184 @@
+package exercises
+
+import stdlib._
+import shapeless.HNil
+
+import org.scalatest.Spec
+import org.scalatest.prop.Checkers
+
+// FIXME: get rid of this if possible
+import org.scalacheck.Shapeless._
+
+class AssertsSpec extends Spec with Checkers {
+  def `scalatest asserts` = {
+    check(Test.success(Asserts.scalaTestAsserts _, true :: HNil))
```

Comment:
I'd be more explicit in the function name and call it testSuccess most people would import Test._ and then become more ambiguous just calling success